.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
```

Comment:
🎉 Woo, I think this is the missing piece when working on https://github.com/holic/web3-scaffold/issues/14